repo/packages/N/nexus/4/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "packagingVersion": "3.0",
+  "name": "nexus",
+  "version": "3.2.1",
```

Comment:
There is a 3.4.0 version available. My team is about to embark on a project to automate the updates for this kind of change but until then we could update in this PR or I can submit a separate PR. Not sure your preference as separate PRs require a new folder in the tree.